Bugfix/move styled footer component to resolve build issue

Urjikoo opened the pull request:
Initially the new styled footer component `CustomFooter` was initialized in app component because of that the build on github was failing. Therefore it's now initialized right before app component. 

## Types of changes
<!--
  What types of changes does your code introduce? Put an `x` in all the boxes
  that apply:
-->
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality
      to change)

## Checklist:
<!--
  Go over all the following points, and put an `x` in all the boxes that apply.
-->
- [X] I have run `eslint` on the code
- [ ] I have added JSDoc for all of my code (where applicable)
- [ ] I have added tests to cover my changes.

## Priority:
- [X] Normal <!-- New piece of functionality -->
- [ ] High <!-- Critical bug requiring urgent review -->

## Related Issues:
<!--
  Add the issue number in below that this PR is intended to address.

  Also mention any issues that this PR is related to, and if possible,
  provide more information regarding the relationship to the issues in
  question (i.e does the PR fix the issue, is it designed to fix another bug
  that is similar to the issue etc.)
-->
Fixes #___

<!--
  Attribution:
  PR Template adapted from: https://github.com/h5bp/html5-boilerplate/blob/master/.github/PULL_REQUEST_TEMPLATE.md
-->

Urjikoo committed:
moving CustomFooter from app component
moved CustomFooter to resolve build issue

natalynyu reviewed (changes requested):
As Jonathan suggested in the sprint planning meeting, we should probably move this `CustomFooter` component away from the `App` file. I think it would make sense to create a new file in the `src/client/components/layout` directory.
<img width="262" alt="image" src="https://user-images.githubusercontent.com/29589689/230979177-586b21f1-d247-40e4-94c3-e956ab1dc3ea.png">

Just as a note, you will need to export in the `index.ts` file like we've done for the other components in the directory:
<img width="510" alt="image" src="https://user-images.githubusercontent.com/29589689/230979397-8ef711d3-aa07-4568-84b5-6a675698a6a6.png">

Urjikoo committed:
Moved CustomFooter component to it's new file from app.tsx
created a new file in layout and moved the styled footer component from app.tsx to resolve build issues.

Urjikoo committed:
Fixing import order for CustomFooter

Urjikoo committed:
Adding CustomFooter file to the commit
CustomFooter file was created and imported to app.tsx however the file wasn't added to initial commit which resulted a
build  issue.

natalynyu reviewed (approved):
Looks great! Approved.
